### PR TITLE
Update Auth Options

### LINKED ISSUE #386: Update references to deprecated authd options and consider the new ones.
As part of epic [#9550](https://github.com/wazuh/wazuh/issues/9550) it is required to add new configuration options to the authd module to decide what actions should be taken during an agent replacement. That being said, the following configuration files should be modified to remove the deprecated options and include the new option block.

```
<force>
  <enabled>yes|no</enabled>
  <key_mismatch>yes|no</key_mismatch>  
  <disconnected_time enabled="yes"|"no">1h</disconnected_time>  
  <after_registration_time>1h</after_registration_time>  
</force>
```
- manifests/manager.pp
- manifests/params_manager.pp
- templates/fragments/_auth.erb

### PR DESCRIPTION
**Pull request by vcerenu**
# QA Standard pull request template

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

---

## Closing/Related Issues
Closes #386 

## Test plan (required)
- [x] I have tested the pipeline on Jenkins. (Mandatory)
https://ci.wazuh.info/view/Tests/job/Test_service/3273/
https://ci.wazuh.info/view/Tests/job/Test_service/3278

## Target Jenkins
- [x] Production
- [x] Staging

Are there dependencies with other repositories? (Remove unmark checks)
- [x] No

Are there dependencies with other pipelines? (Remove unmark checks)
- [x] No

## Description/Notes
Hello team,

This PR update the last parameters introduced in 4.3.0 auth section inside ossec.conf.

### CODE REVIEW

**Review by lastranikos — APPROVED** [review]
LGTM

File: manifests/params_manager.pp [hunk 1]
```diff
@@ -199,8 +199,10 @@
       $ossec_auth_disabled                             = 'no'
       $ossec_auth_port                                 = 1515
       $ossec_auth_use_source_ip                        = 'yes'
-      $ossec_auth_force_insert                         = 'yes'
-      $ossec_auth_force_time                           = 0
+      $ossec_auth_force_enabled                        = 'yes'
+      $ossec_auth_force_key_mismatch                   = 'yes'
+      $ossec_auth_force_disc_time                      = '5s'
+      $ossec_auth_force_after_reg_time                 = '5s'
```
Comment (okynos) [inline manifests/params_manager.pp:205]: Set it to `1h`

**Review by okynos — CHANGES_REQUESTED** [review]
Minor change.

File: manifests/params_manager.pp (hunk 1 above)
Comment (vcerenu) [reply manifests/params_manager.pp]: Done

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**